src/bento/util/template.js

```diff
@@ -0,0 +1,17 @@
+import {escapeCssSelectorIdent} from '#core/dom/css-selectors';
+
+/**
+ * @param {HTMLElement} element
+ * @return {HTMLTemplateElement|null}
+ */
+export function getTemplate(element) {
+  if (element.hasAttribute('template')) {
+    const id = element.getAttribute('template');
```

Comment:
What about components that can utilize more than 1 template?
Eg. the `amp-list` component has a `template` for items, but can also use `wrapper-template` and `load-more-template` for other UI needs.  What's a good way to support these in Bento?

Reply:
Let's talk about this offline.

Reply:
How about just taking the attribute?
```suggestion
export function getTemplate(element, attribute = 'template') {
  if (element.hasAttribute(attribute)) {
    const id = element.getAttribute(attribute);
```